Entire test suite fails with Django 3.1b1 on Windows

When trying to look at the `url()` deprecation I installed Django 3.1b1. Every test fails with this version of Django.

Not sure if this should be raised here or as a ticket for Django. @carltongibson ?

I'm using Python 3.8.3 (latest release)

```
Python 3.8.3 (tags/v3.8.3:6f8c832, May 13 2020, 22:37:02) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

```

_________________________ ERROR at setup of TestSlugRelatedField.test_overriding_get_queryset _________________________
..\venv\django-rest-framework\lib\site-packages\django\db\backends\utils.py:82: in _execute
    return self.cursor.execute(sql)
..\venv\django-rest-framework\lib\site-packages\django\db\backends\sqlite3\base.py:413: in execute
    return Database.Cursor.execute(self, query)
E   sqlite3.OperationalError: no such function: JSON_VALID

The above exception was the direct cause of the following exception:
..\venv\django-rest-framework\lib\site-packages\pytest_django\plugin.py:534: in _django_setup_unittest
    request.getfixturevalue("django_db_setup")
..\venv\django-rest-framework\lib\site-packages\pytest_django\fixtures.py:105: in django_db_setup
    db_cfg = setup_databases(
..\venv\django-rest-framework\lib\site-packages\django\test\utils.py:170: in setup_databases
    connection.creation.create_test_db(
..\venv\django-rest-framework\lib\site-packages\django\db\backends\base\creation.py:65: in create_test_db
    call_command(
..\venv\django-rest-framework\lib\site-packages\django\core\management\__init__.py:168: in call_command
    return command.execute(*args, **defaults)
..\venv\django-rest-framework\lib\site-packages\django\core\management\base.py:371: in execute
    output = self.handle(*args, **options)
..\venv\django-rest-framework\lib\site-packages\django\core\management\base.py:85: in wrapped
    res = handle_func(*args, **kwargs)
..\venv\django-rest-framework\lib\site-packages\django\core\management\commands\migrate.py:214: in handle
    self.sync_apps(connection, executor.loader.unmigrated_apps)
..\venv\django-rest-framework\lib\site-packages\django\core\management\commands\migrate.py:348: in sync_apps
    editor.create_model(model)
..\venv\django-rest-framework\lib\site-packages\django\db\backends\base\schema.py:324: in create_model
    self.execute(sql, params or None)
..\venv\django-rest-framework\lib\site-packages\django\db\backends\base\schema.py:142: in execute
    cursor.execute(sql, params)
..\venv\django-rest-framework\lib\site-packages\django\db\backends\utils.py:66: in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
..\venv\django-rest-framework\lib\site-packages\django\db\backends\utils.py:75: in _execute_with_wrappers
    return executor(sql, params, many, context)
..\venv\django-rest-framework\lib\site-packages\django\db\backends\utils.py:84: in _execute
    return self.cursor.execute(sql, params)
..\venv\django-rest-framework\lib\site-packages\django\db\utils.py:90: in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
..\venv\django-rest-framework\lib\site-packages\django\db\backends\utils.py:82: in _execute
    return self.cursor.execute(sql)
..\venv\django-rest-framework\lib\site-packages\django\db\backends\sqlite3\base.py:413: in execute
    return Database.Cursor.execute(self, query)
E   django.db.utils.OperationalError: no such function: JSON_VALID
```

--- a/tests/test_serializer_nested.py
+++ b/tests/test_serializer_nested.py
@@ -311,6 +311,9 @@ if postgres_fields:
         """Model declaring a postgres JSONField"""
         data = postgres_fields.JSONField()
 
+        class Meta:
+            required_db_features = {'supports_json_field'}
+
 
 @pytest.mark.skipif(not postgres_fields, reason='psycopg2 is not installed')
 class TestNestedNonRelationalFieldWrite:

